spec/helpers/rejection_helper_spec.rb

```diff
@@ -11,7 +11,8 @@
 
   describe "#rejection_description" do
     it "gives rejection description from the locale file" do
-      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue.</p>')
+      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue. We cannot accept a new petition when we already have one about a very similar issue.</p>
+<p>You are more likely to get action on this issue if you sign and share a single petition.</p>')
```

Comment:
I wonder whether changing this assertion to use a `match` on a partial string would be more readable, e.g:

``` ruby
expect(helper.rejection_description(code)).to match(%r[<p>There’s already a petition about this issue</p>])
```

Just nit-picking - I'll leave it up to you :smile: